src/Dependencies/Collections/SegmentedArray`1.cs

```diff
@@ -7,7 +7,6 @@
 using System.Collections.Generic;
 using System.Reflection.Emit;
 using System.Runtime.CompilerServices;
-using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.Shared.Collections
```

Comment:
>Shared [](start = 33, length = 6)

I'd omit the "Shared" subnamespace. It doesn't add much value, imo.

Reply:
➡️ Removed